go.mod

```diff
@@ -15,7 +15,10 @@ require (
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.15.0
+<<<<<<< HEAD
```

Comment:
@KlavsKlavsen can you take a look at this? It's causing CI to fail. Your changes should only touch files in the charts/ folder, right?

Reply:
My bad - merge of upstream master.. rebased now.